community_images/fluent-bit/ironbank/image.yml

```diff
@@ -0,0 +1,44 @@
+name: fluent-bit-ib
+official_name: Fluent-Bit Ironbank
+official_website: https://fluentbit.io/
+source_image_provider: Platform One
+source_image_repo: registry1.dso.mil/ironbank/opensource/fluent/fluent-bit
+source_image_repo_link: https://registry1.dso.mil/harbor/projects/3/repositories/opensource%2Ffluent%2fluent-bit
+source_image_readme: https://repo1.dso.mil/dsop/opensource/fluent/fluent-bit/-/blob/development/README.md
+rf_docker_link: rapidfort/fluent-bit-ib
+image_workflow_name: fluent-bit_ironbank
+github_location: fluent-bit/ironbank
+report_url: https://us01.rapidfort.com/app/community/imageinfo/registry1.dso.mil%2Fironbank%2Fopensource%2Ffluent%2Ffluent-bit
+usage_instructions: |
+  # install fluent-bit, using docker run
+  $ docker run -it --name Fluent-Bit rapidfort/fluent-bit-ib
```

Comment:
Please verify if this would work, do we need port mapping, conf, env variables, volumes in docker run.

Reply:
It works, we don't need any of it